packages/core/base/src/services/crossmintModalService.ts

```diff
@@ -118,9 +118,12 @@ export function crossmintModalService({
             if (showOverlay) {
                 addLoadingOverlay();
             }
-        } else {
-            setConnecting(false);
-            console.error("Failed to open popup window");
+            return;
+        }
+        setConnecting(false);
+        const newTab = window.open(url, "_blank");
```

Comment:
I think when a new tab opens, we dont need for the button to be `Connecting...` nor the overlay to be set IMO, since you are in a different context.

 Its a bit weird that you have to close the recently opened tab for the overlay to disappear...